com.civicrm.hrjobroles/CRM/Hrjobroles/BAO/Query.php

```diff
@@ -37,9 +37,18 @@ class CRM_Hrjobroles_BAO_Query extends CRM_Contact_BAO_Query_Interface {
         '1' => '%'
     );
 
+    /**
+     * @var array A list of fields available while importing/exporting
+     */
+    private $hrjobRoleFields;
```

Comment:
i think it's better to give this an initial empty value , something like :

private $hrjobRoleFields = array();